brightwind/analyse/analyse.py

```diff
@@ -1425,7 +1431,12 @@ def time_continuity_gaps(data):
 
     continuity = pd.DataFrame({'Date From': indexes.values.flatten()[:-1],
                                'Date To': indexes.values.flatten()[1:]})
-    continuity['Days Lost'] = (continuity['Date To'] - continuity['Date From']) / pd.Timedelta('1 days')
+    continuity['Days Lost'] = (continuity['Date To'] - continuity['Date From'])
```

Comment:
I can see why you removed the pd.Timedelta from this line so you can filter out gaps smaller than a certain length. I think the variable should be renamed though so as to match what it represents i.e. it doesn't represent 'Days Lost' anymore as that is calculated below.